cypress/support/validations/schemaValidation.js

```diff
@@ -250,3 +220,54 @@ function removeSetInMethodName(apiName) {
   }
   return apiName.split('.')[0] + '.' + updatedMethod;
 }
+
+/**
+ * @module schemaValidation
+ * @function validateLifecycleSchema
+ * @description validate lifecycle response against corresponding schema and return schema validation result
+ * @param {string} response - lifecycle response
+ * @param {string} methodSchema - schema against which validation needs to be performed
+ * @example
+ * validateLifecycleSchema(response, methodSchema)
+ */
+Cypress.Commands.add('validateLifecycleSchema', (response, methodSchema) => {
+  let validationResult;
+  const schemaMapResult = validator.validate(response, methodSchema);
+  if (schemaMapResult.errors.length > 0 || response === undefined) {
+    validationResult = {
+      status: CONSTANTS.FAIL,
+      schemaValidationResult: schemaMapResult,
+    };
+  } else {
+    validationResult = {
+      status: CONSTANTS.PASS,
+      schemaValidationResult: schemaMapResult,
+    };
+  }
+  return validationResult;
+});
+
+/**
+ * @module schemaValidation
+ * @function updateLifecycleResponse
+ * @description update raw lifecycle response from 3rd party app with schema validation fields
+ * @param {string} response - lifecycle response
+ * @param {string} method - method name
+ * @example
+ * updateLifecycleResponse(response, method)
+ */
```

Comment:
why do we need to additional functions for lifecycle methods ? 
Ideally , all of them should be using the same function used by 1st party app calls and 3rd party app calls glue code . 
https://github.com/rdkcentral/firebolt-certification-suite/pull/43/files#diff-0f45d4dbc3a685533c52006d47c2c307b6bac8501a5b543a56f1f0021ae779abR166